www/js/app.js

```diff
@@ -63,6 +63,6 @@ angular.module('starter', ['ionic', 'starter.controllers', 'ng-token-auth', 'ngC
   $urlRouterProvider.otherwise('/app/playlists');
 
   $authProvider.configure({
-    apiUrl: 'http://localhost:3000/api/v1' || 'https://rails-rockabye.herokuapp.com/'
+    apiUrl: 'https://rails-rockabye.herokuapp.com/api/v1' || 'http://localhost:3000/api/v1'
```

Comment:
usually the way we difference between heroku (production) and localhost (development) is by having an environment variable defined in heroku (ENV) and then checking if it exists, otherwise refer to localhost. In this case, it will always return the heroku URL for your API.

Since you are in ionic, you won't have an ENV variable to access by default, so you may be have to include it like this: http://jbavari.github.io/blog/2014/08/23/managing-environment-variables-for-your-ionic-application/

ping me if you need help with this